graphics/src/MeshManager.cc

```diff
@@ -1648,3 +1660,82 @@ void MeshManager::SetAssimpEnvs()
     this->dataPtr->forceAssimp = true;
   }
 }
+
+//////////////////////////////////////////////////
+std::vector<SubMesh>
+MeshManager::ConvexDecomposition(const SubMesh *_subMesh,
+                                 std::size_t _maxConvexHulls,
+                                 std::size_t _voxelResolution)
+{
+  std::vector<SubMesh> decomposed;
+
+  if (!_subMesh)
+    return decomposed;
+
+  auto vertexCount = _subMesh->VertexCount();
+  auto indexCount = _subMesh->IndexCount();
+  auto triangleCount = indexCount / 3u;
+
+  float *points = new float[vertexCount * 3u];
```

Comment:
Consider using `std::vector` or `std::unique_ptr`.

Reply:
changed to `std::vector`. [a6c5a2f](https://github.com/gazebosim/gz-common/pull/585/commits/a6c5a2f6c565e4ec636c1f492631312b470a5696)